python/lancedb/remote/client.py

```diff
@@ -109,6 +109,7 @@ async def post(
         params: Optional[Dict[str, Any]] = None,
         content_type: Optional[str] = None,
         deserialize: Callable = lambda resp: resp.json(),
+        request_id: Optional[str] = None,
```

Comment:
why is that

Reply:
we already have request id processing from header. Our middleware will use client provided request id instead of generating one if the client has set `x-request-id`